src/main/java/com/bootcamp/demo/model/PaymentDetailsDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.bootcamp.demo.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class PaymentDetailsDTO {
+    private Card card;
```

Comment:
Please analyze if all card details are required for a placing a payment, or if only the card id would suffice.